resources/docker-compose/apim/docker-compose.yaml

```diff
@@ -85,7 +85,7 @@ services:
       - ENFORCER_REGION=UNKNOWN
       - XDS_MAX_MSG_SIZE=4194304
       - XDS_MAX_RETRIES=3
-      - JAVA_OPTS=${JAVA_OPTS} -Dhttpclient.hostnameVerifier=AllowAll
+      - JAVA_OPTS=${JAVA_OPTS} -javaagent:/home/wso2/lib/org.jacoco.agent-0.8.8-runtime.jar=destfile=/home/wso2/lib/dropins/aggregate.exec,append=true -Dhttpclient.hostnameVerifier=AllowAll
```

Comment:
These lines should not be modified. We do not need jacoco stuff to run for users. It is just required when running integration tests.